errbot/backends/telegram_messenger.py

```diff
@@ -182,6 +183,7 @@ def __init__(self, config):
 
         identity = config.BOT_IDENTITY
         self.token = identity.get('token', None)
+        self.botname = identity.get('username', None)
```

Comment:
I believe you could get the bot's username through the [getMe](https://core.telegram.org/bots/api#getme) API call, so there would be no need to require the user to set it manually.